realm/realm-library/src/main/java/io/realm/internal/modules/CompositeMediator.java

```diff
@@ -115,6 +115,14 @@ public String getTableName(Class<? extends RealmObject> clazz) {
         return mediator.createDetachedCopy(realmObject, maxDepth, cache);
     }
 
+    @Override
+    public boolean transformerApplied() {
+        if (mediators.isEmpty()) {
```

Comment:
Is this accurate? I mean, this will crash with a "Transform not running", but wouldn't the cause more likely be that the annotation processor didn't ran?